app/views/layouts/nav/staff/_program_subnav.html.haml

```diff
@@ -0,0 +1,19 @@
+.navbar.navbar-default.program-subnav
+  .container-fluid
+    %ul.nav.navbar-nav
```

Comment:
Couple questions:

Do you want the balance-scale icon on both Review Proposals and Proposals?

What's the reasoning behind the order of subnav items? The user flow is Proposals --> Selection --> Sessions